src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -220,36 +243,133 @@ public CreateArticleModalComponentObject clickRedLink(int i) {
+    return this;
   }
 
-  public void compareFontSizes(WebElement firstElement, WebElement secondElement) {
+  public PortableInfoboxPageObject verifyDivsNotAppearingInHeader(int index) {
+    Assertion.assertNotEquals(groupHeadersWrappers.get(index).getTagName(), "div");
+    return this;
+  }
+
+  public void verifyElementsNotWrappedByDivs() {
+    verifyDivsNotAppearing(imageWrapper);
+    verifyDivsNotAppearing(titleWrapper);
+    verifyDivsNotAppearing(groupHeadersWrappers.get(0));
+  }
+
+  public PortableInfoboxPageObject compareFontSizes(WebElement firstElement, WebElement secondElement) {
     String firstFontSize = firstElement.getCssValue("font-size");
     String secondFontSize = secondElement.getCssValue("font-size");
     Assertion.assertEquals(firstFontSize, secondFontSize);
+    return this;
+  }
+
+  public void compareListsFontSizes() {
+    compareFontSizes(itemValue, orderedElementList.get(0));
+    compareFontSizes(itemValue, unorderedElementList.get(0));
+  }
+
+  public void compareHorizontalGroupFontSizes() {
+    compareFontSizes(itemLabel, horizontalItemLabel);
+    compareFontSizes(itemValue, horizontalItemValue);
   }
 
   public String getInfoboxContent() {
     return infoboxLayout.getText();
   }
 
-  public void verifyEmptyTags(String infoboxContent) {
-    Assertion.assertStringContains(infoboxContent, "Default");
+  public PortableInfoboxPageObject compareFontSizesBetweenHorizontalItemLabelAndItemLabel() {
+    compareFontSizes(horizontalItemLabel, itemLabel);
+    return this;
   }
+
+  public PortableInfoboxPageObject compareFontSizesBetweenHorizontalItemValueAndItemValue() {
+    compareFontSizes(horizontalItemValue, itemValue);
+    return this;
+  }
+
+  public PortableInfoboxPageObject compareFontSizesBetweenItemValueAndOrderedListItem(int index) {
+    compareFontSizes(itemValue, orderedElementList.get(index));
+    return this;
+  }
+
+  public PortableInfoboxPageObject compareFontSizesBetweenItemValueAndUnorderedListItem(int index) {
+    compareFontSizes(itemValue, unorderedElementList.get(index));
+    return this;
+  }
+
+  public PortableInfoboxPageObject verifyEmptyTags() {
+    Assertion.assertStringContains(infoboxLayout.getText(), "Default");
+    return this;
+  }
+
+  public PortableInfoboxPageObject verifyGroupHeaderPadding(int index) {
+    verifyPadding(getGroupHeader(index));
+    return this;
+  }
+
+  public void verifyNavigationPadding() {
+    verifyPadding(navigationElements.get(1));
+  }
+
+  public PortableInfoboxPageObject open(String articleTitle) {
+    getUrl(urlBuilder.getUrlForWiki(Configuration.getWikiName()) + URLsContent.WIKI_DIR
+           + articleTitle);
+    return this;
+  }
+
+  public PortableInfoboxPageObject areBoldElementsMoreThanOne() {
+    Assertion.assertTrue(boldElements.size() > 0);
+    return this;
+  }
+
+  public PortableInfoboxPageObject areItalicElementsMoreThanOne() {
+    Assertion.assertTrue(italicElements.size() > 0);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)